src/reports/components/report-sections/no-failed-instances-congrats.scss

```diff
@@ -8,14 +8,14 @@
 }
 
 .report-congrats-head {
-    font-size: 48px;
+    font-size: 28px;
```

Comment:
Fer suggested a smaller font size in the linked issue (16 or 14px). Should we use that instead?

Reply:
Fixed in iteration 2